### PR TITLE
fix food issue *temporary fix* and added hacky fast transfer *much faster*

### PR DESCRIPTION
**Pull request by NycuRO**

### CODE REVIEW

File: src/main/java/org/itxtech/synapseapi/utils/DataPacketEidReplacer.java [hunk 1]
```diff
@@ -112,38 +118,19 @@ private static EntityMetadata replaceMetadata(EntityMetadata data, long from, lo
...
-            } catch (Exception e) {
-                MainLogger.getLogger().error("Exception while replacing metadata '" + key + "'", e);
+                data.putLong(key, to);
             }
         }
 
         return data;
     }
 
-    @SuppressWarnings("rawtypes")
```
Comment (PetteriM1) [inline src/main/java/org/itxtech/synapseapi/utils/DataPacketEidReplacer.java]: Why?

File: src/main/java/org/itxtech/synapseapi/SynapsePlayer.java [hunk 2]
```diff
@@ -73,7 +74,7 @@ public SynapseEntry getSynapseEntry() {
     }
 
     @Override
-    protected void processLogin() {
+    public void processLogin() {
```
Comment (PetteriM1) [inline src/main/java/org/itxtech/synapseapi/SynapsePlayer.java]: Why?

File: src/main/java/org/itxtech/synapseapi/SynapseAPI.java [hunk 3]
```diff
@@ -45,6 +45,17 @@ public void onLoad() {
 
     @Override
     public void onEnable() {
+        this.getServer().getScheduler().scheduleRepeatingTask(new cn.nukkit.scheduler.Task() {
+            @Override
+            public void onRun(int i) {
+                for (Player player : Server.getInstance().getOnlinePlayers().values()) {
+                    player.setHealth(player.getHealth());
```
Comment (PetteriM1) [inline src/main/java/org/itxtech/synapseapi/SynapseAPI.java]: Isn't food bar the only thing we have problems with?

File: src/main/java/org/itxtech/synapseapi/SynapsePlayer.java (hunk 2 above)
Comment (NycuRO) [reply src/main/java/org/itxtech/synapseapi/SynapsePlayer.java]: just for my testing, you can make protected back

File: src/main/java/org/itxtech/synapseapi/SynapseAPI.java (hunk 3 above)
Comment (NycuRO) [reply src/main/java/org/itxtech/synapseapi/SynapseAPI.java]: nop, sometime heal too

File: src/main/java/org/itxtech/synapseapi/utils/DataPacketEidReplacer.java (hunk 1 above)
Comment (NycuRO) [reply src/main/java/org/itxtech/synapseapi/utils/DataPacketEidReplacer.java]: you can add back

File: src/main/java/org/itxtech/synapseapi/SynapseAPI.java (hunk 3 above)
Comment (PetteriM1) [reply src/main/java/org/itxtech/synapseapi/SynapseAPI.java]: How about experience?

File: src/main/java/org/itxtech/synapseapi/utils/DataPacketEidReplacer.java (hunk 1 above)
Comment (PetteriM1) [reply src/main/java/org/itxtech/synapseapi/utils/DataPacketEidReplacer.java]: What's the point of change something that will be changed back in next commit?

**Pull request closed without merging**